### PR TITLE
Refactor insight's chart component

### PR DESCRIPTION
**Pull request by cakecatz**
**What this PR does / why we need it**:

- Update recharts.js 
- Fix chart design a little
  - remove YAxis line
  - Show year at the chart label
- Change chart message for no data

image

image

image

**Which issue(s) this PR fixes**:

Fixes #

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Comment by nghialv** [comment]
@cakecatz Can you fix the test?

**Comment by cakecatz** [comment]
@nghialv Fixed!

File: pkg/app/web/src/components/deployment-frequency-chart.tsx [hunk 1]
```diff
@@ -12,26 +12,36 @@ import {
...
   data: InsightDataPoint.AsObject[];
 }
 
+const tickFormatter = (time: number | string): string =>
+  dayjs(time).format("MMM DD");
+
+const labelFormatter = (time: number | string): string =>
+  dayjs(time).format("YYYY MMM DD");
+
+const NO_DATA_TEXT = "No data is available.";
```
Comment (khanhtc1202) [inline pkg/app/web/src/components/deployment-frequency-chart.tsx:40]: should move to constant directory 👀

Comment (khanhtc1202) [reply pkg/app/web/src/components/deployment-frequency-chart.tsx:40]: in the second thought, I guess this value is encapsulated with this component only, so the current place is LGTM 👍

**Review by nghialv — APPROVED** [review]

**Pull request merged**